tests/integration/t4r/test_pytorch_backend.py

```diff
@@ -134,33 +95,69 @@ def test_serve_t4r_with_torchscript(tmpdir):
     # Build a simple Ensemble graph
     # ===========================================
 
-    output_schema = Schema([ColumnSchema("output", dtype=np.float32)])
-
-    torch_op = merlin_yoochoose_schema.column_names >> PredictPyTorch(
-        traced_model, merlin_yoochoose_schema, output_schema
+    input_schema = model.input_schema(max_session_len)
+    output_schema = model.output_schema()
+
+    # add batch dim to int shapes
+    for col_schema in input_schema:
+        shape = col_schema.properties.get("shape")
+        if shape and isinstance(shape, int):
+            input_schema[col_schema.name] = col_schema.with_properties({"shape": [-1, shape]})
+
+    # add batch dim to int shapes
+    for col_schema in output_schema:
+        shape = col_schema.properties.get("shape")
+        if shape and isinstance(shape, int):
+            col_schema = col_schema.with_properties({"shape": [-1, shape]})
+            if shape > 1:
+                col_schema = ColumnSchema(
+                    col_schema.name,
+                    dtype=col_schema.dtype,
+                    properties={
+                        **col_schema.properties,
+                        "value_count": {"min": shape, "max": shape},
+                    },
+                    tags=col_schema.tags,
+                    is_list=True,
+                    is_ragged=False,
+                )
+            output_schema[col_schema.name] = col_schema
+
+    torch_op = input_schema.column_names >> PredictPyTorch(
+        traced_model, input_schema, output_schema
     )
 
-    ensemble = Ensemble(torch_op, merlin_yoochoose_schema)
+    ensemble = Ensemble(torch_op, input_schema)
     ens_config, node_configs = ensemble.export(str(tmpdir))
 
     # ===========================================
     # Convert training data to Triton format
     # ===========================================
 
     df_cols = {}
+    missing_cols = []
     for name, tensor in torch_yoochoose_like.items():
-        df_cols[name] = tensor.cpu().numpy().astype(merlin_yoochoose_schema[name].dtype)
+        if name not in input_schema.column_names:
+            missing_cols.append(name)
```

Comment:
There's an issue here with columns missing from the input_schema. comment in T4R here: https://github.com/NVIDIA-Merlin/Transformers4Rec/pull/507#discussion_r1007902048